sktime/forecasting/model_selection/_tune.py

```diff
@@ -195,9 +126,9 @@ def check_is_fitted(self, method_name=None):
                     "attribute" % (type(self).__name__, method_name)
                 )
             else:
-                self.best_forecaster_.check_is_fitted()
+                self._get_delegate().check_is_fitted()
 
-    def _fit(self, y, X=None, fh=None):
+    def fit(self, y, X=None, fh=None):
```

Comment:
why do you rename here to `fit()`? This means you overwrite the parent `fit()` and I think we should not do so 🤔

Reply:
So, it was actually intentional that I overwrote the parent `fit()`, though I no longer do so now. :)  Long story short - the overwriting was a way to get around the delegation of `fit()`, but turns out this solution requires some sort of hacky workaround anyway.... so now this is `_fit()` again, and we no longer attempt to delegate `BaseGricCV`'s `fit()` - though we do still delegate the other functions!  If you get a chance I would appreciate your thoughts on the new version!  :) Thank you for flagging this, it was kind of a subtle point, and I think the new version that doesn't do this is cleaner.